VocabularyTrainer/Home Screen/HomeViewController.swift

```diff
@@ -0,0 +1,249 @@
+//
+//  HomeViewController.swift
+//  VocabularyTrainer
+//
+//  Created by skrr on 18.12.22.
+//  Copyright © 2022 mic. All rights reserved.
+//
+
+import Foundation
```

Comment:
import Foundation was needed?

Reply:
good catch :) no it's not needed.

Probably a leftover from when I started the file 🙈

Reply:
removed it here: 3562f2d8fdb5f668d44473b564f64b5f216d0bac